apps/judicial-system/backend/src/app/modules/case/models/case.model.ts

```diff
@@ -863,4 +863,14 @@ export class Case extends Model<Case> {
   })
   @ApiProperty()
   caseResentExplanation?: string
+
+  /**********
+   * Indicates whether the case has been archived - optional
+   **********/
+  @Column({
+    type: DataType.BOOLEAN,
+    allowNull: true,
```

Comment:
Should we not allow a null value here?
```suggestion
    allowNull: false,
```

Reply:
Agreed, that is in line with the database migration.